core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationWarnings.java

```diff
@@ -33,9 +33,19 @@
 public final class ConfigurationWarnings extends BaseConfigurationWarnings {
 	private static ConfigurationWarnings self = null;
 	private Configuration activeConfiguration = null;
-	public static final String SQL_INJECTION_SUPPRESS_KEY = "warnings.suppress.sqlInjections";
-	public static final String DEPRECATION_SUPPRESS_KEY="warnings.suppress.deprecated";
-	public static final String DEFAULT_VALUE_SUPPRESS_KEY = "warnings.suppress.defaultvalue";
+
+	public static void addGlobalWarning(Logger log, String msg, SuppressKeys suppressionKey, ClassLoader classLoader) {
+		if(suppressionKey == null) {
+			addGlobalWarning(log, msg, null);
+		} else if(!isSuppressed(suppressionKey, null, classLoader)) {
+			addGlobalWarning(log, msg, null);
+			// provide suppression hint as info
+			if(log.isInfoEnabled() && suppressionKey.isAllowGlobalSuppression()) {
+				String msgWithHint = msg + ". This warning can be suppressed by setting the property '"+suppressionKey.getKey()+"=true'";
+				log.info(msgWithHint);
+			}
```

Comment:
Now msg is logged twice: first without the hint in addGlobalWarning(), then here with the hint.
This should not happen. The message should be logged only once, (with or without hint, depending on log.isInfoEnabled())